altair/api.py

```diff
@@ -72,9 +72,31 @@ def to_dict(self):
 
 
 class Scale(BaseObject):
-    # TODO: Fill out
+    """Scale object that represents the Scale property common to: X, Y, Size, Color"""
+    # TODO: Supported types for type
+    # TODO: Supported types for reverse
+    # TODO: Supported types for zero
+    # TODO: Supported types for nice
+
+    type = T.Enum(['linear', 'log', 'pow', 'sqrt', 'quantile'], default_value='linear')
+    reverse = T.Bool(False)
+    zero = T.Bool(True)
+    nice = T.Enum(['second', 'minute', 'hour', 'day', 'week', 'month', 'year'], allow_none=True)
+    useRawDomain = T.Bool(default_value=None, allow_none=True)
 
-    pass
+
+class ScaleColor(Scale):
```

Comment:
Do  they have this object in the spec? If not, I would `ColorScale` make more sense?